src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/CodeStyle/CSharpCodeStyleOptions.cs

```diff
@@ -321,6 +321,12 @@ private static Option2<CodeStyleOption2<AddImportPlacement>> CreateUsingDirectiv
             EditorConfigStorageLocation.ForBoolCodeStyleOption("csharp_style_allow_blank_line_after_colon_in_constructor_initializer_experimental", CodeStyleOptions2.TrueWithSilentEnforcement),
             new RoamingProfileStorageLocation("TextEditor.CSharp.Specific.AllowBlankLineAfterColonInConstructorInitializer"));
 
+        public static Option2<CodeStyleOption2<bool>> AllowBlankLineAfterTokenInConditionalExpression { get; } = CreateOption(
+            CSharpCodeStyleOptionGroups.NewLinePreferences, nameof(AllowBlankLineAfterTokenInConditionalExpression),
+            CSharpIdeCodeStyleOptions.Default.AllowBlankLineAfterTokenInConditionalExpression,
+            EditorConfigStorageLocation.ForBoolCodeStyleOption("csharp_style_allow_blank_line_after_token_in_conditional_expression_experimental", CodeStyleOptions2.TrueWithSilentEnforcement),
+            new RoamingProfileStorageLocation("TextEditor.CSharp.Specific.AllowBlankLineAfterTokenInConditionalExpression"));
```

Comment:
The roaming option is probably not needed.

Reply:
Following hte pattern of hte rest of the newline options.

Reply:
@CyrusNajmabadi It seems the others are in VS UI and AutomationObject, in this case the roaming location is needed. Do you plan to add this new option in VS UI and AutomationObject as well?

Reply:
i'll do whatever makes this in-line with the rest of the newline options :)